source/draw/DrawState.ooc

```diff
@@ -38,6 +38,8 @@ DrawState: cover {
 	_sourceNormalized := FloatBox2D new(0.0f, 0.0f, 1.0f, 1.0f)
 	_focalLengthNormalized := 0.0f // Relative to image width
 	targetTransform := FloatTransform2D identity // 2D scaling done after projection matrix but before w division
+	nearPlane: Float = 0.01f
+	farPlane: Float = 12500.0f
```

Comment:
If not set, the uninitialized data will be given to `_createProjection` and override the default values with uninitialized values (possibly NaN causing a once in a million crash).